src/js/components/app.js

```diff
@@ -34,6 +35,34 @@ const activationPath = '/activate';
 const timeout = 900000; // 15 minutes idle time
 const cookies = new Cookies();
 
+const reducePalette =
+  prefix =>
+  (accu, [key, value]) => {
+    if (value instanceof Object) {
+      return {
+        ...accu,
+        ...Object.entries(value).reduce(reducePalette(`${prefix}-${key}`), {})
+      };
+    } else {
+      accu[`${prefix}-${key}`] = value;
+    }
+    return accu;
+  };
+
+const cssVariables = ({ palette }) => {
+  const muiVariables = Object.entries(palette).reduce(reducePalette('--mui'), {});
+  return {
+    '@global': {
+      ':root': {
+        ...muiVariables,
+        '--mui-overlay': palette.grey[400]
```

Comment:
the 400 is basically using the definition... defined in [here](https://github.com/mendersoftware/gui/blob/master/src/js/themes/Mender/light.js#L8-L19)
using a constant representation would be `palette.grey['400']`, but then I would think `palette.grey.400` should work, but that won't work in javascript

as to why not to define this with a more granular name: the number approach is supposed to target the different shades of a color (most often grey, due to the popularity of using shades of grey), since the naming depends so much on subjective perception/ the display you are using/ ... in CSS there's e.g. `dimgrey` and `darkgrey` - guess which is darker?

Reply:
I bet dimgrey is darker!

Reply:
so... on my display: yes!

Reply:
ha! you see, not only I know my colour consts, I also am familiar with your display :>